core/templates/dev/head/pages/exploration_editor/editor_tab/StateSolutionDirective.js

```diff
@@ -0,0 +1,248 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for solution.
+ */
+oppia.directive('stateSolution', [
```

Comment:
stateSolutionEditor

Reply:
Similar comment here as well. There is a separate solution editor directive. This just takes care of displaying the title and other info related to solutions in the main editor window and call the solution editor directive when required.

Reply:
Done, made it stateSolutionEditor.